### PR TITLE
intial model creation with some tests more tests and functionality to…

### PR DESCRIPTION
**Pull request by kfless12**
Add the orders model and the join table with adjusted columns. Some testing was implemented and more will be added along with a hook for order completion.

### CODE REVIEW

File: server/db/index.js [hunk 1]
```diff
@@ -1,2 +1,10 @@
 //Everything comes together here (models from hooks)
 //export db, models
+
+const db = require('./db')
+const Order = require('./models/Order')
+const Product = require('./models/Product')
+const ProductOrders = require('./models/ProductOrders')
+
```
Comment (alejvillanueva) [inline server/db/index.js:8]: These models should be imported from hooks.js (even if you dont have any hooks set up yet)

File: server/db/relationships.js [hunk 2]
```diff
@@ -1 +1,8 @@
-//Models imported from Model js
+const { Sequelize } = require('sequelize')
+const Order = require('./models/Order')
+const Product = require('./models/Product')
+
+
+
```
Comment (alejvillanueva) [inline server/db/relationships.js:6]: You need export the models here as well and import them into hooks.

File: server/main.js [hunk 3]
```diff
@@ -1,8 +1,11 @@
 const app = require('./server')
 const PORT = process.env.PORT || 3000
+const { db } = require('./db/index')
 
 //const require db
 
+db.sync()
+
```
Comment (alejvillanueva) [inline server/main.js:8]: db.sync() should be in a try catch inside of app.listen():

```suggestion
app.listen(PORT, async () => {
    try {
        await db.sync()
        console.log(`Listening on ${PORT}`)
    } catch (error) {
        console.log(error)
    }
})

```

Comment (kfless12) [reply server/main.js]: updated

File: server/db/db.js [hunk 4]
```diff
@@ -1,2 +1,10 @@
...
+const Sequelize = require('sequelize')
+
+const conn = new Sequelize(
+    process.env.DATABASE_URL || 'postgres://localhost:5432/snacker',
+    {
+        logging: false,
+    }
+)
+
+module.exports = conn
```
Comment (alejvillanueva) [inline server/db/db.js]: You are importing db throughout your other files but export conn here. Not sure if the naming could cause issues here.

**Review by alejvillanueva — APPROVED** [review]
Awesome changes! Approving and will merge with mine

**Pull request merged**